app/about_us/our_crew/page.js

```diff
@@ -0,0 +1,114 @@
+import styles from "./page.module.css";
+
+const OurCrew = () => {
+  // TASK - React 1 week 1
+  // Create the "Our Crew section"
+  // Use the descriptions provided in /app/about_us/README.md
+  // Use the pictures from /public/crew
+  // Some inspiration ideas found in /data/inspiration_about_us
+  return (
+    <>
+      <div className={styles.ourCrew}>
+        <p>OUR CREW </p>
+        <h1>Who are we</h1>
+        <p>
+          Our crew is the heart and soul of Galactica. We are a diverse team of
+          seasoned space explorers, engineers, and visionaries who are united by
+          a common goal: to make space travel accessible and exciting for all.
+        </p>
+      </div>
+      <div className={styles.members}>
+        <section className={styles.memberSection}>
+          <img
+            src={`/crew/Sarah.png`}
+            alt="Sarah_Image"
```

Comment:
This `alt="..."` text is not providing any extra info for the non-sighted users, so you can either change it or remove it.
(We are already having the crew members name in the `<h3>` below.)
- More info on alt tag: https://www.w3.org/WAI/tutorials/images/
- Videos about Accessibility testing, and how people use assistive technologies:
  - Short intro video - https://www.youtube.com/watch?v=HE2R86EZPMA
  - Longer and detailed video from a non-sighted user - https://www.youtube.com/watch?v=OUDV1gqs9GA

Reply:
I wrote that it is picture of Sarah, in case due to some issue picture is not displaying the alt will explain what was there